### ISSUE TITLE
Advertise lightningNodeKey + raidenAddress

### ISSUE THREAD

**Issue by moshababo** [ExchangeUnion/xud#156]
Advertise `lightningNodeKey` + `raidenAddress` on handshaking.
On the receiver side it will be used for opening channels & swaps operations.

**Comment by kilrau** [comment]
Where did you think this will be stored?

**Comment by sangaman** [comment]
The `nodes` table. This shouldn't be too hard and it'd be nice to have in the alpha release.

**Comment by kilrau** [comment]
Continuation of https://github.com/ExchangeUnion/xud/pull/243

Flow:
1. Discover LND + Raiden address on startup
2. Exchange LND + Raiden address on handshake with peers
3. Don't store persistently, it should be on every handshake (enables updates)

**Comment by sangaman** [comment]
This functionality actually already exists, implemented in #362 and #243.

```typescript
export type HandshakeState = {
  version: string;
  nodePubKey: string;
  addresses?: Address[];
  pairs: string[];
  raidenAddress?: string;
  lndbtcPubKey?: string;
  lndltcPubKey?: string;
};
```
